text/0148-sampling-probability.md

```diff
@@ -0,0 +1,794 @@
+despite their effective count.  We can use this to record error
+exemplars, for example, even when they are not selected by the
+Sampler.
+
+## Proposed Tracing specification
+
+For the standard OpenTelemetry Span Sampler implementations to support
+a range of probability sampling schemes, this document recommends the
+use of a Span attribute named `sampling.adjusted_count` to encode an
+unbiased adjusted count computed by a Sampler reflecting the whole 
+population of spans.
+
+The value the `sampling.adjusted_count` attribute under this proposal
+MUST be an unbiased estimate of the total population count represented
+by the individual event.
+
+### Suggested text
+
+After
+[Sampler](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#sampler)
+and before [Builtin
+Samplers](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#built-in-samplers),
+a new section will introduce the term "adjusted count" and relate it
+to inclusion probability.  For example:
+
+```markdown
+### Probability sampling
+
+Probability Samplers are Samplers that output statistically unbiased
+inclusion probability.  Inclusion probability in the context of
+tracing is the *effective* probability of the Sampler returning
+`RECORD_AND_SAMPLE` when invoked, which can be decided locally or 
+derived from the context when the [W3C Trace Context is-sampled
+flag](https://www.w3.org/TR/trace-context/#sampled-flag) is in use.
+
+#### Adjusted count span attribute
+
+The recommended way to convey inclusion probability *for events*
+sampled in OpenTelemetry is in the form of an **adjusted count**,
+which is the reciprocal (i.e., mathematical inverse function) of
+inclusion probability.
+
+The implied goal of probability sampling is to support estimating the
+count of spans in the population using the spans that were sampled.
+The adjusted count associated with a Span is the expected value of the
+number of identical Spans within the population that each Span
+represents.  Probability samplers SHOULD ensure the samples they
+compute are unbiased, which implies that the sum
+of adjusted counts in the sample approximates the true count of spans in the
+population.
+
+The adjusted count in an unbiased probability sampling scheme SHOULD
+be expressed using a Span attribute named `sampling.adjusted_count`
+when it represents the whole population of events.  Adjusted count 
+attributes MAY be integer or floating-point values.
+
+#### Inclusion probability tracestate value
+
+The recommended way to convey sampling probability *for contexts* in
+OpenTelemetry is through the W3C Trace Context tracestate using the
+key `head_probability`.
+
+Probability Samplers MAY encode the effective sampling inclusion
+probability using tracestate, for the context that was in effect when
+the W3C is-sampled bit was set.  The tracestate field SHOULD be set in
```

Comment:
> The tracestate field SHOULD be set in

This needs to be qualified, so that SHOULD is interpreted conditionally, e.g.

> **If the inclusion probability is propagated**, it SHOULD be represented with subkey `headprob` within the field `otel`... For example:
```
tracestate: otel=headprob:0.1234
```